### PR TITLE
Sort and extend cpell.json

### PR DESCRIPTION
**Pull request by hunger**

### CODE REVIEW

File: cspell.json [hunk 1]
```diff
@@ -51,34 +43,53 @@
...
     ],
     "words": [
+        "analysing",
+        "Bezier",
+        "cbindgen",
+        "cmake",
+        "colspan",
+        "datastructures",
         "Goffart",
+        "gradians",
```
Comment (tronical) [inline cspell.json:61]: This looks like a … typo :-). Where did you encounter it?

Comment (tronical) [reply cspell.json:61]: Nevermind, my bad spelling - it’s correct of course :)

File: cspell.json [hunk 2]
```diff
@@ -51,34 +43,53 @@
...
-                "SUBDIR",
                 "rustup",
+                "SUBDIR",
                 "Yocto"
             ],
             "ignoreRegExpList": []
         }
     ],
     "words": [
+        "analysing",
```
Comment (tronical) [inline cspell.json:54]:
```suggestion
        "analysing", // deliberate choice of British spelling here
```

Comment (tronical) [reply cspell.json:54]: Maybe worth a comment or changing to US spelling?

Comment (hunger) [reply cspell.json:54]: Oh right! That's british spelling. I totally did not have that on the radar and was stumped why this was flagged as wrong!

Let's stick with US spelling, I'll remove this again.

**Review by tronical — APPROVED** [review]

**Pull request merged**